common/strings.c

```diff
@@ -119,46 +119,51 @@ TDNFStringSepCount(
 
 uint32_t
 TDNFSplitStringToArray(
-    char *pszBuf,
+    const char *pszBuf,
     char *pszSep,
     char ***pppszTokens
     )
 {
     uint32_t dwError = 0;
-    char *pszTok = NULL;
-    char *pszState = NULL;
     char **ppszToks = NULL;
-    size_t nCount = 0;
-    size_t nIndex = 0;
+    const char *p, *p0;
```

Comment:
reduce the scope of p0 variable and please try to use a better name

Reply:
Variable name is perfect.